addons/point_of_sale/models/pos_session.py

```diff
@@ -23,7 +23,7 @@ def _confirm_orders(self):
             for order in session.order_ids.filtered(lambda o: o.state != 'done'):
                 if order.state not in ('paid', 'invoiced'):
                     raise UserError(_("You cannot confirm all orders of this session, because they have not the 'paid' status"))
-                order.signal_workflow('done')
+                order.action_pos_order_done()
```

Comment:
@Icallhimtest 
Are you now the new responsible for PoS?
As already said, https://github.com/odoo/odoo/pull/15178 does it a better way...it does not make sense to test against invoiced orders if they are paid...btw. this is another regression in v10 not yet fixed (see the PR text)